examples/server/tests/unit/test_embedding.py

```diff
@@ -194,3 +196,32 @@ def test_embedding_usage_multiple():
     assert res.status_code == 200
     assert res.body['usage']['prompt_tokens'] == res.body['usage']['total_tokens']
     assert res.body['usage']['prompt_tokens'] == 2 * 9
+
+
+def test_embedding_openai_library_base64():
+    server.start()
+    test_input = "Test base64 embedding output"
+
+    res = server.make_request("POST", "/v1/embeddings", data={
+        "input": test_input,
+        "encoding_format": "base64"
+    })
+
+    assert res.status_code == 200
+    assert "data" in res.body
+    assert len(res.body["data"]) == 1
+
+    embedding_data = res.body["data"][0]
+    assert "embedding" in embedding_data
+    assert isinstance(embedding_data["embedding"], str)
+
+    # Verify embedding is valid base64
+    try:
+        decoded = base64.b64decode(embedding_data["embedding"])
+        # Verify decoded data can be converted back to float array
+        float_count = len(decoded) // 4  # 4 bytes per float
+        floats = struct.unpack(f'{float_count}f', decoded)
+        assert len(floats) > 0
+        assert all(isinstance(x, float) for x in floats)
+    except Exception as e:
+        pytest.fail(f"Invalid base64 format: {str(e)}")
```

Comment:
It's a bad practice to use try..catch inside a test, because if one of the `assert` fails, it won't let you know it happens at exactly which line of code.

Reply:
@ngxson, thanks for the hint. my bad, I forgot to remove it.